pkg/ccl/kvccl/kvfollowerreadsccl/testdata/boundedstaleness/single_row

```diff
@@ -212,7 +212,7 @@ ALTER TABLE t2 ADD COLUMN new_col INT
 query idx=2
 SELECT * FROM t2 AS OF SYSTEM TIME with_min_timestamp(now() - '10s', true) WHERE pk = 2
 ----
-pq: descriptor not found
+pq: referenced descriptor ID 55: descriptor not found
```

Comment:
This is one of the very rare side-effects of having changed error handling somewhat in the new `catkv` package.

Reply:
I'm pleased with it.